services/frontend/src/store/index.js

```diff
@@ -36,29 +39,79 @@ export default new Vuex.Store({
     },
     setPatients(state, patients) {
       state.patients = patients
+    },
+    setRecordingID(state, recordingID) {
+      state.analysis.recordingID = recordingID
+    },
+    setAnalysisID(state, id) {
+      state.analysis.analysisID = id
     }
   },
   actions: {
+    //! those are initial implementation, change when implementing real feature
     //TODO: change email if needed
     async getPatient(state, email) {
-      let patient = await axios.get('user/'+email) // yeah this doesn't work rn 
+      let patient = await api.getPatient(state.state.user.token, email) // yeah this doesn't work rn 
```

Comment:
"state.state" - this is a bad naming scheme

Reply:
done, changed to contex.state